dpp/src/main/java/org/dashj/platform/dashpay/BlockchainIdentity.kt

```diff
@@ -2006,16 +2006,16 @@ class BlockchainIdentity {
             val xpub = decryptExtendedPublicKey(
                 contactRequest.encryptedPublicKey,
                 contactIdentity,
-                otherContactKeyIndex = contactRequest.recipientKeyIndex, // other
-                myKeyIndex = contactRequest.senderKeyIndex, // mine
+                otherContactKeyIndex = contactRequest.senderKeyIndex, // other
+                myKeyIndex = contactRequest.recipientKeyIndex, // mine
```

Comment:
These changes were to undo some changes from a few weeks ago that had sender and receiver backwards